Gemfile.lock

```diff
@@ -134,7 +134,7 @@ GEM
     bootsnap (1.5.1)
       msgpack (~> 1.0)
     brakeman (4.10.0)
-    browser (4.2.0)
+    browser (5.2.0)
```

Comment:
This one feels a bit weird. Normally if we use something directly, I'd advocate for moving it to the `Gemfile` and not relying on the transitive dependency, but here we only use it to configure something related to the gem that depends on it, so that seems wrong too.

Reply:
Yeah, this part is a bit weird